lib/core/msdf-sys/src/lib.rs

```diff
@@ -0,0 +1,163 @@
+extern crate wasm_bindgen;
+#[macro_use] extern crate slice_as_array;
+
+use wasm_bindgen::prelude::*;
+use js_sys::*;
+
+const MAX_MSDF_SIZE : usize = 32;
+pub const MSDF_CHANNELS_COUNT : usize = 3;
+const MSDF_DATA_SIZE : usize = MAX_MSDF_SIZE*MAX_MSDF_SIZE*MSDF_CHANNELS_COUNT;
+
+pub struct FontHandle {
+    handle: JsValue
+}
+
+pub struct MSDF {
+    pub width  : usize,
+    pub height : usize,
+    pub data   : [f32;MSDF_DATA_SIZE]
+}
+
+pub struct MSDFParameters {
+    width                         : usize,
+    height                        : usize,
+    edge_coloring_angle_threshold : f64,
+    range                         : f64,
+    scale                         : (f64, f64),
+    translate                     : (f64, f64),
+    edge_threshold                : f64,
+    overlap_support               : bool
+}
+
+#[wasm_bindgen(module = "msdfgen_wasm.js")]
+extern {
+
+fn ccall(
```

Comment:
this name is not descriptive. Lets either rename it or add docs here. In fact docs should be added to other methods as well (See the code guide for rust - docs section)

Reply:
I can't choose its name, because this is function provided by emscripten. But I'll add comment here